docs/configuration.md

```diff
@@ -37,12 +43,96 @@ If you are running these applications without Docker, just make sure that any en
 the environment where the application is running.
 
 If you are doing development on a url other than `localhost` or `localhost:5000`, you need to update the
-API_DEV_HOSTNAME value to your development hostname:port, e.g. `myhost:8000` or `api.testserver.net`.
-DEV_MODE should be `true`.
-
-If you are deploying to a custom domain (not pol.is) than you need to update both the API_PROD_HOSTNAME and
-DOMAIN_OVERRIDE values to your custom hostname (omitting http(s):// protocol).
-DEV_MODE should be `false`.
+**`API_DEV_HOSTNAME`** value to your development hostname:port, e.g. `myhost:8000` or `api.testserver.net`.
+**`DEV_MODE`** should be `true`.
+
+If you are deploying to a custom domain (not pol.is) then you need to update both the **`API_PROD_HOSTNAME`** and
+**`DOMAIN_OVERRIDE`** values to your custom hostname (omitting http(s):// protocol).
+**`DEV_MODE`** should be `false`.
+
+### General Settings
+
+- **`ADMIN_UIDS`** an array of user UUIDs for site admins. These users will have moderator capabilities on all conversations hosted on the site.
+- **`EMAIL_TRANSPORT_TYPES`** comma-separated list of email services to use (see [Email Transports](#email-transports) below)
+- **`GIT_HASH`** Set programatically using `git rev-parse HEAD` (e.g. in Makefile) to tag docker container versions and other release assets. Can be left blank.
+- **`MATH_ENV`** Set to prod (default), preprod, or dev. In cases where a single math service is used for multiple environments, this value is used by the API service to request the correct data.
```

Comment:
It's actually for cases where a single _database_ is used to serve multiple environments (with separate server+math-worker for each); Should also probably note that this pattern is no longer recommended now that there is better support for instantiating a separate dedicated db.